addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -164,6 +164,10 @@
             }
         }
 
+        .o_field_many2manytags.avatar {
+            margin-left: 6px;
```

Comment:
I would rather use `rem` as unit. I think that usually we are using 2rem in odoo to space items. Could you double check if that renders ok?